bhclub.html

```diff
@@ -52,7 +53,26 @@ <h1>Events</h1>
             <h1>Teams</h1>
         </section>
         <section id="contact">
-            <h1>Contact</h1>
+            <div>
+                <h1>Contact</h1>
+                <p>Have any questions? Feel free to reach out to us at <a href="mailto:hello@brickhack.org">hello@brickhack.org</a></p>
```

Comment:
nit: missing punctuation
```suggestion
                <p>Have any questions? Feel free to reach out to us at <a href="mailto:hello@brickhack.org">hello@brickhack.org</a>.</p>
```

Reply:
I'm not sure if adding extra punctuation to the end of an email is a good idea.
The period in the next line is actually getting changed to a colon; design is having me swap around some of the elements in this section, implementing those soon